lib/views/desktop/settings.dart

```diff
@@ -53,30 +62,147 @@ class _SettingsState extends SettingsShared<Settings> {
                         ),
                         Column(
                           children: [
-                            Heading(
-                              title: '$CURRENT_SAVE_DESTINATION',
-                              textAlign: TextAlign.center,
-                              marginTop: 110.0.h,
-                              path: path,
-                              textStyle: Theme.of(context).textTheme.headline6,
-                              key: Key(SETTINGS_SCREEN_HEADING),
-                            ),
                             SizedBox(
-                              height: 45.0.h,
+                              height: 16.0,
                             ),
                             DTButtonWithBackground(
                               onPressed: selectSaveDestination,
                               title: SELECT_A_FOLDER,
                               width: 150.0,
                               disabled: false,
-                            )
+                            ),
+                            Container(
```

Comment:
Hard to read and understand containers part, but not sure if is possible to group components/box in better code visibility.